catalogs/catalog-jdbc-doris/src/main/java/org/apache/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -170,13 +170,12 @@ protected String generateCreateTableSql(
   }
 
   private Map<String, String> appendNecessaryProperties(Map<String, String> properties) {
-    Map<String, String> resultMap;
     if (properties == null) {
-      resultMap = new HashMap<>();
-    } else {
-      resultMap = new HashMap<>(properties);
+      return new HashMap<>();
     }
 
+    Map<String, String> resultMap = new HashMap<>(properties);
```

Comment:
I believe replacing `properties` with `resultMap` in L184 can solve the problem, your changes I think can't address it.

Reply:
My thoughts was, the problem in L184 `!properties.containsKey(REPLICATION_FACTOR` would be `properties` can be null and NPE could occur, so in L173 `properties == null` we check if it is NULL and return a empty Map right way.

Reply:
@yuqi1129 , have made the changes, will resolve the convo after you check.